ktxspec.adoc

```diff
@@ -1145,6 +1175,20 @@ submitting the data to a Basis encoder then the DFD must not
 have a sample with a channelType that indicates it is alpha.
 ====
 
+Basis GPU Photo::
```

Comment:
Should KTX skip the 6-byte header of the GPU Photo stream? Neither ID not dimensions are technically needed when the stream is used in KTX.

Reply:
Good question. Seems reasonable to skip them. @richgel999 what do you think?

Reply:
@MarkCallow Basis wiki now calls this format "ASTC HDR 6x6 Intermediate". We should be careful with naming throughout the spec.

Reply:
I am not going to push commits with trial balloons for naming. Once we resolve the naming in our offline discussion I will update this PR.

Reply:
If my suggestion on UASTC 6x6 supercompression given in https://github.com/KhronosGroup/KTX-Specification/pull/216/files#r1966377934 is okay then this section is not needed any more.

We still need an answer from @richgel999 about the 6-byte header. The answer will affect the bitstream description we will add to KDFS.

Reply:
One more detail from the source code: width and height of that "Intermediate" format are limited to 32768.

Reply:
Besides the header, the decoder also expects each row of blocks to start with a known pair of bytes and an image to end with the end marker.

Reply:
This thread is still not explicitly resolved. KTX files must not use that 6-byte header.

Reply:
@richgel can we skip the 6-byte header in KTX?

Reply:
>@richgel can we skip the 6-byte header in KTX?

This header mirrors the common .ASTC file format from ARM, but cut down for a 6x6 2D block size (which is the only size this particular supercompression format supports). It makes the embedded ASTC data independent of its container (.basis or .KTX2, or none). It's an existing well-known pattern from the ASTC world, which is why it's used here. The compressed blocks follow the header. 
 
<img width="774" height="324" alt="image" src="https://github.com/user-attachments/assets/2b79ff1c-fb4b-4987-842c-620452997580" />

https://developer.arm.com/-/media/Arm%20Developer%20Community/PDF/ASTC_User_Guide_102162_0001_01.pdf

Reply:
I am against incorporating this header because it will appear for _every_ image in the KTX file and making sure it is correct and doesn't conflict with the KTX header and DFD information will, as @lexaknyazev says, add implementation complexity especially in the validator. Since we have no idea what users will do when parsing these files we can't afford a mismatch, if both are present. No other format carried by KTX, including the ASTC formats, has the image type and size repeated in each image.

I will add a note to Appendix F to say that the header described in the reference should not be included and when we add the format to KDFS it will not feature the header. @richgel please modify your KTX support accordingly.